### PR TITLE
Drn 291 journey changes

### PR DESCRIPTION
**Pull request by mattrobertsky**

### CODE REVIEW

File: app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala [hunk 1]
```diff
@@ -102,21 +99,19 @@ class JourneyController @Inject()(
...
-  )(
-    implicit request: AuthorisedRequest[AnyContent]
-  ) =
-    setState(state).map {_=>
-      Logger.debug(message)
-      redirect
-    }
+
+
+    private def updateStateAndRedirect(
```
Comment (alexflav23) [inline app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala:104]: Could probably be `private[this]`.

File: app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/ReturningUserController.scala [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  with AuthorisedFunctions {
+
+
+  val interpreter = DRInterpreter(appConfig, this, messagesApi)
+  import interpreter._
+  implicit val persistence: PersistenceEngine[AuthorisedRequest[AnyContent]] =
+    returningUserSave4LaterPersistence.persistence
+
+
+  def returningUserAction(targetId: String): Action[AnyContent] = authorisedAction.async {
```
Comment (alexflav23) [inline app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/ReturningUserController.scala:60]: Should we tag this somehow, or are we happy with `String`?

Comment (hmrclt) [reply app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/ReturningUserController.scala:60]: This is the part of the URL coming in from the route that uniform uses. What would you tag it as?

File: app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala (hunk 1 above)
Comment (hmrclt) [reply app/uk/gov/hmrc/disguisedremunerationfrontend/controllers/JourneyController.scala:104]: I'm not sure I understand why `private[this]` would be preferable to `private` here?

**Pull request merged**